.bumpversion.cfg

```diff
@@ -0,0 +1,17 @@
+[bumpversion]
+current_version = 2.0.4
+commit = True
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(?:-(?P<prerel>rc)\.(?P<prerelversion>\d+))?
```

Comment:
How about adding `tag = True`? Nice to have these tags in the commit history.

Reply:
or maybe just for the final release version

Reply:
I added the `--tag` flag to the command in the Makefile when we bump to the version without the `rc` suffix. Hope that is cool with you!